### PR TITLE
Update pw_tutorial.rst

### PR DESCRIPTION
**Pull request by ltalirz**
fix https://github.com/aiidateam/aiida-quantumespresso/issues/766

Start with job submission example to demonstrate the goal of this tutorial (+ allow people who can immediately grasp how things work to skip most of it).

@mbercx How does this look?

Will still need to test the script, but this is what I had in mind.

### CODE REVIEW

**Comment by ltalirz** [comment]
Script is tested; now also includes missing instructions for loading the pseudopotentials

File: docs/source/user_guide/get_started/examples/pw_short_example.py [hunk 1]
```diff
@@ -8,55 +8,26 @@
...
-
-###############################
-# Set your values here
-codename = 'pw-6.3@TheHive'
-pseudo_family = 'SSSP_efficiency'
-###############################
-
-code = Code.get_from_string(codename)
-builder = code.get_builder()
+builder = orm.Code.get_from_string('pw-6.3@TheHive').get_builder()
```
Comment (sphuber) [inline docs/source/user_guide/get_started/examples/pw_short_example.py:16]:
```suggestion
builder = orm.load_code('pw-6.3@TheHive').get_builder()
```

File: docs/source/user_guide/get_started/examples/pw_tutorial.rst [hunk 2]
```diff
@@ -61,9 +59,15 @@ if the following input is not clear to you, please refer to the Quantum ESPRESSO
           0.0000000000       4.0000000000       0.0000000000
           0.0000000000       0.0000000000       4.0000000000
 
-Without AiiDA, not only you would have to prepare 'manually' this file,
-but also prepare the scheduler submission script, send everything on the cluster, etc.
-We are going instead to prepare everything in a more programmatic way.
+Using the ``aiida-quantumespresso`` plugin, you can want to submit the same calculation via the following Python script:
```
Comment (mbercx) [inline docs/source/user_guide/get_started/examples/pw_tutorial.rst:62]:
```suggestion
Using the ``aiida-quantumespresso`` plugin, you can submit the same calculation via the following Python script:
```

**Review by mbercx — CHANGES_REQUESTED** [review]
Thanks @ltalirz, just one small correction.

**Pull request merged**